src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/TraceLogging/TraceLoggingEventSource.cs

```diff
@@ -187,7 +192,12 @@ public unsafe void Write(string? eventName, EventSourceOptions options)
         /// public instance properties of data will be written recursively to
         /// create the fields of the event.
         /// </param>
+#if !ES_BUILD_STANDALONE
+        [RequiresUnreferencedCode(EventSourceRequiresUnreferenceMessage)]
+        public unsafe void Write<[DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.PublicProperties)] T>(
```

Comment:
Is there a reason that the DynamicallyAccessedMembers attribute appears on this overload of Write\<T> but doesn't appear on the overloads below?

Reply:
That was my bad, thanks for spotting it!